core/src/test/scala/query/interpolator/RuntimeCQLInterpolatorSpec.scala

```diff
@@ -48,15 +48,14 @@ class RuntimeCQLInterpolatorSpec extends WordSpec with Matchers {
       val id: Int      = 1
       val name: String = "username"
 
-      val result: Try[(String, List[OutputValue])] =
+      val result: (String, List[SerializableValueBy[Int]]) =
```

Comment:
`val (query, serializableValues) = cql"SELECT * FROM users WHERE id = $id AND name = $name"`